haptic_mpc:
    darci_sim:
        # Skin topic lists. Each should be publishing a TaxelArray message
        skin_list:
          none: []
          fabric: ['/elbowclevis_LEFT_tactile_sensor/taxels/forces', '/wrist_LEFT_tactile_sensor/taxels/forces', '/handmount_LEFT_tactile_sensor/taxels/forces']
          #fabric: ['/wrist_LEFT_tactile_sensor/taxels/forces', '/handmount_LEFT_tactile_sensor/taxels/forces', '/flipper_tactile_sensor/taxels/forces']

        # Frames used for TF. torso_frame is the one the arm is attached to, inertial frame is to get the torso position in the world
        torso_frame: '/torso_lift_link'
        inertial_frame: '/world'
        end_effector_frame: 'end_effector_LEFT'

        # Joint limits.
        # Should be equal in number and in the same order as the number of links in the arm (eg 7 for the PR2, starting at the shoulder). 
        # Should be specified in DEGREES.
        joint_limits:
          right:
            max: [200, 150, 85, 133, 195, 30, 30]
            min: [-80, -25, -85, 0, -20, -30, -30]
          left:
            max: [200, 25, 85, 133, 20, 30, 30]
            min: [-80, -150, -85, 0, -195, -30, -30]

    #    joint_limits:
    #      right:
    #        max: [200, 150, 85, 133, 195, 60, 60]
    #        min: [-80, -25, -85, 0, -20, -60, -60]
    #      left:
    #        max: [200, 25, 85, 133, 20, 60, 60]
    #        min: [-80, -150, -85, 0, -195, -60, -60]
